docs/add-api.md

```diff
@@ -72,7 +73,7 @@ Please manually edit the file created at /notes-app/amplify/backend/api/notesapp
 ? Press enter to continue
 ```
 
-9. Then push the changes to the cloud. Accept all the defaults and continue.
+9. Then push the changes to the cloud. Accept all the defaults and continue. This process will create the API in the cloud, and generate the code in your project you need to use it.
```

Comment:
To reinforce that all the somewhat-intimidating looking defaults are doing something we want !